connexion/resolver.py

```diff
@@ -98,33 +98,52 @@ def resolve_operation_id_using_rest_semantics(self, operation):
 
         :type operation: connexion.operation.Operation
         """
-        path_match = re.search(
-            '^/?(?P<resource_name>([\w\-](?<!/))*)(?P<trailing_slash>/*)(?P<extended_path>.*)$', operation.path
-        )
+        path_fragments = operation.path.split('/')
+
+        tail_parameter = ''
+        has_trailing_slash = False
+        if path_fragments:
+            tail = path_fragments[-1]
+            if tail == '':
+                has_trailing_slash = True
+            elif tail.startswith('{'):
```

Comment:
What about intermediate path segments with path parameters, like `/path/{id}/subpath`?

Reply:
Nevermind. I see it later on. Deleting comment.